Content.Shared/Stealth/SharedStealthSystem.cs

```diff
@@ -54,10 +58,30 @@ public virtual void SetEnabled(EntityUid uid, bool value, StealthComponent? comp
         if (!Resolve(uid, ref component, false) || component.Enabled == value)
             return;
 
+        if (!_mobState.IsDead(uid) && value == false)
+            return;
+
         component.Enabled = value;
         Dirty(component);
     }
```

Comment:
This is going to block all stealth being turned off on an alive mob which it shouldn't be doing.

Reply:
done